packages/react/package.json

```
@@ -50,6 +50,7 @@
     "build:storybook:visual-testing": "storybook build",
     "build:components.json": "tsx script/components-json/build.ts",
     "build:precompile-color-schemes": "tsx script/precompile-color-schemes.ts",
+    "figma:sync": "export $(cat .env | xargs) && npx figma connect publish",
```

Is it possible to do this without the local `.env` file and have folks testing it pass the ENV variable explicit?

```suggestion
    "figma:sync": "figma connect publish",
```